numpy/_core/src/multiarray/nditer_impl.h

```diff
@@ -121,8 +121,6 @@
 #define NPY_OP_ITFLAG_CAST         0x0004
 /* The operand never needs buffering */
 #define NPY_OP_ITFLAG_BUFNEVER     0x0008
-/* The operand is aligned */
```

Comment:
I'd tend to either leave a comment here about why there is a gap, or just be blunt and move all the flags down - since this is internal, that should be completely fine.